pkg/controller/mizarcontrollers/util.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 Authors of Arktos.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mizarcontrollers
+
+type KeyWithEventType struct {
```

Comment:
this small file seems to able to incorporate in other files, as it only has very limited stuff.

Reply:
There are some planned func coming. For example, I need a function to transfer from pod object to pod grpc contract object.